### PR TITLE
Add Italian Localization

### PR DESCRIPTION
**Pull request by davidepanato**
Ciao Till, I have created Italian localization.
Davide

### CODE REVIEW

File: components/philips_coffee_machine/localization.h [hunk 1]
```diff
@@ -54,6 +54,54 @@ namespace esphome
...
+
+        const std::string state_ground_coffee_selected = "Selezionato Caffè Premacinato";
+        const std::string state_coffee_programming_mode = "Selezionata Modalità programmazione Caffè";
+        const std::string state_coffee_selected = "Selezionato Caffè";
+        const std::string state_coffee_2x_selected = "Selezionati 2 Caffè";
+        const std::string state_coffee_brewing = "Erogazione Caffè";
+        const std::string state_coffee_2x_brewing = "Erogazione 2 Caffè";
+
+        const std::string state_ground_espresso_selected = "Selezionate Espresso Premacinato";
+        const std::string state_espresso_programming_mode = "Selezionata Modalità programmazione Espresso"";
```
Comment (TillFleisch) [inline components/philips_coffee_machine/localization.h:76]: This line has a duplicate " at the end

**Comment by TillFleisch** [comment]
The main __init__.py needs an update as well, see [here](https://github.com/TillFleisch/ESPHome-Philips-Smart-Coffee/blob/866fb171cdeea1e0dfbdf0aad2cdfc57f3bc1bf7/components/philips_coffee_machine/__init__.py#L30)

File: components/philips_coffee_machine/localization.h (hunk 1 above)
Comment (davidepanato) [reply components/philips_coffee_machine/localization.h]: Fixed!

**Comment by davidepanato** [comment]
I have also modified __init__.py and readme.MD. Sorry but I do not have full environment to build at the moment :(

**Review by TillFleisch — APPROVED** [review]
@davidepanato Thank you for adding Italian language support! :)

**Pull request merged**